### PR TITLE
Add warning to OCC as well in case another PHP version is used there

### PR DESCRIPTION
**Pull request by LukasReschke**
In regard to a recent rant on social media where users complained about us to have broken the command line tool. Guess reading the documentation is hard…

cc @blizzz

### CODE REVIEW

File: console.php [hunk 1]
```diff
@@ -28,6 +28,14 @@
 
 define('OC_CONSOLE', 1);
 
+// Show warning if a PHP version below 5.4.0 is used, this has to happen here
+// because base.php will already use 5.4 syntax.
+if (version_compare(PHP_VERSION, '5.4.0') === -1) {
+	echo "This version of ownCloud requires at least PHP 5.4.0\n";
```
Comment (blizzz) [inline console.php]: use PHP_EOL for line break

**Comment by MorrisJobke** [comment]
Works :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
jenkins is green -> merge

**Pull request merged**